Felienne/hedy
# [FIX] Altered time-out to 20 seconds if either turtle or sleep

**Description**
Currently we have a time-out implemented on the front-end preventing programs from running either to run or running indefinitely. This value is set to 3 seconds without the turtle and to 20 seconds with the turtle. With the introduction of the `sleep` command the 3 seconds limit is easily exceeded. Therefore in this PR we set the time-out to 20 seconds for either the turtle or the sleep command, fixing this issue.

**Fix for**
This PR fixes #1671 

**How to test**
Run a program with the `sleep` keyword that runs longer than 3 seconds but shorter than 20 seconds. Notice that the code executes correctly. For example:

```
print hallo wereld!
sleep 10
print duurt lang dit...
```

**Checklist**

If you're unsure about any of these, don't hesitate to ask. We're here to help!
